doc/contributing/advocacy-ambasador-program.md

```diff
@@ -0,0 +1,93 @@
+# Advocacy Ambassador program
+
+Node.js aims to support and encourage developers and developer
+advocates to promote Node.js as a project and to help communicate
+key messages to their constituencies.
+
+The ambassador program does that by:
+
+* Maintaining an up-to-date set of messages that it would like ambassadors
+  to promote. These will be documented in the nodejs/ambassadors repository.
+* Maintaining an up-to-date set of topics on which additional content
+  would benefit the community. These will be documented in the
+  nodejs/ambassadors repository.
+* Providing a space for collaborators to share new topics/features they
+  are working on, when they would be happy for ambassadors to help share and get
+  early feedback.
+* Helping ambassadors promote content that aligns with the
+  messages and topics defined through social media and other channels.
+* Advocating for ambassadors to be part of the OpenJS speakers bureau, even if the
+  ambassador is not otherwise an active member of the project itself.
```

Comment:
Maybe I'm failing to separate "the ambassador program" from "ambassadors". Does this effectively mean that ambassadors advocate for ambassadors to be part of the OpenJS speakers bureau?

Reply:
What's inteded is that the project, through the abassador project "Advocates for ambassadors to be part of the OpenJS Speakers bureau, ..."